ebu_tt_live/xsd1.1/ebutt_datatypes.xsd

```diff
@@ -88,9 +98,14 @@ Please note that the EBU-TT XML Schema is a helping document and NOT normative b
 			<xs:pattern value="[+-]?[0-9]+(\.[0-9]+)?(h|m|s|ms)"/>
 		</xs:restriction>
 	</xs:simpleType>
+	<xs:simpleType name="limitedClockTimingType">
+		<xs:restriction base="xs:string">
+			<xs:pattern value="[0-9][0-9]:[0-5][0-9]:[0-5][0-9]:[0-9][0-9]"/>
```

Comment:
This seems to have frames at the end instead of fractions of seconds - I think only smpteTimingType allows frames. Also, the limitedClockTimingType is used in clockTimingType and that restricts the seconds to the range 00-60. Taken together I think this expression should probably be `[0-9][0-9]:[0-5][0-9]:([0-5][0-9]|60)(\.[0-9]+)?`.